test/config/config.exs

```diff
@@ -0,0 +1,57 @@
+use Mix.Config
+
+system = :nerves_system_rpi0
+platform = "rpi0"
+arch = "arm"
+
+test_server = System.get_env("NERVES_TEST_SERVER")
+network_interface = System.get_env("NERVES_NETWORK_INTERFACE") || "usb0"
+websocket_protocol = System.get_env("WEBSOCKET_PROTOCOL") || "ws"
+
+config :bootloader,
+  app: :nerves_system_test,
+  init: [:nerves_runtime, :nerves_network]
+
+config :nerves_system_test, :firmware,
+  public_key: System.get_env("NERVES_FW_PUB_KEY")
+
+config :system_registry, SystemRegistry.TermStorage,
+  path: "/root/system_registry",
+  scopes: [
+    [:config, :network_interface, "wlan0", :ssid],
+    [:config, :network_interface, "wlan0", :psk]
+  ]
+
+config :nerves_network, :default,
```

Comment:
Could you add docs here for why `usb0` and `eth0` are listed and what this does? I wouldn't expect them to be used for the test.